templates/vue/src/components/TapestryFilter.vue

```diff
@@ -110,6 +121,10 @@ export default {
       const tapestryApi = new TapestryApi(wpPostId)
       this.allContributors = await tapestryApi.getAllContributors()
     }
+    this.allStatuses = new Set()
+    for (let node of Object.values(this.nodes)) {
+      this.allStatuses.add(node.status)
+    }
```

Comment:
I'm thinking it might be better to hard code the statuses here since it's only going to be "submitted", "rejected", "draft" or something. Maybe move the status to a constants file?

Reply:
OK hear me out, users will be confused if say "submitted" is an option, but no nodes are "submitted", so it shows all nodes faded out. Unless you want that option to be greyed out if none match - in which case you still have to do this set thing.

Reply:
I think that's fine, since if all nodes are faded out it also explicitly shows that there's no nodes submitted. What do you think @wynnset?

Reply:
I think it's a bit harder to tell when the only valid ones are grandchildren

Reply:
Let's:
- Store the statuses in a constants file
- Still loop through the nodes to find how many nodes have each status 
- Display that number in a bracket next to each status (e.g. "Submitted (0)")
- If the number is equal to 0, let's disable the option so the user cannot select it and also visually grey it out or something